### PR TITLE
Porfolio page mobile events

### PR DESCRIPTION
**Pull request by ijeomaodinko**
This file contains the following files:
The inex.html file, style.css, and main.js files.
It contains linters files.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -75,6 +76,10 @@ h1 {
   margin: 2rem 0 0 4rem;
 }
 
+.up-menu a {
+  color: #fff;
+}
```
Comment (keroloussamy) [inline style.css:81]: - [ ] Your menu is working perfectly, just one thing is you have to make your desktop version match the figma design. Kindly make the navigations items appear on the desktop version, you have to change the `color: #fff;` to make them appear.

Your design
image

Figma design
image

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```
Comment (keroloussamy) [inline .eslintrc.json:25]: - [ ] Great work adding linters, just one thing to make them run on GitHub is you have to update your `linters.yml` with this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml), After that chack that your linters are passing.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ijeomaodinko, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ijeomaodinko, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brytebee** [comment]
Hi, @ijeomaodinko👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



**_You have submitted a project, that has been merged already. Kindly confirm and submit a branch with the correct work for this milestone._**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**